lib/dialects/mssql/query.js

```diff
@@ -211,6 +213,20 @@ class Query extends AbstractQuery {
     });
   }
 
+  handleShowConstraintsQuery(data) {
+    data.splice(0, 1);
+    //Convert snake_case keys to camelCase as its generated by stored procedure
+    for (const constraint of data) {
+      for (const key in constraint) {
+        const camelKey = _.camelCase(key);
+        constraint[camelKey] = constraint[key];
+        delete constraint[key];
```

Comment:
this will put the object in dictionary mode (disable V8 hidden classes). It is always faster to create a new object or set to `undefined`